Update choosing_a_solution.md (#173)
exercism/docs: mentoring/choosing_a_solution.md

### The student's code

On the left hand side of the screen you will see the student's code.
The UI will look something like this:

[TODO: Image of mentor discussion with areas marked].

> 1. The left hand side shows you the student's code.

Reviewer: ```suggestion
1. The main part of the left side contains the student's code.
```

   By default you'll be seeing their most recent iteration.

>    If they've submitted multiple iterations, you can switch through these using the numbers in circles at the bottom. If they've only submitted one iteration, you won't see those icons.

Reviewer: ```suggestion
   If they've submitted multiple iterations, you can switch through these using the numbers in circles at the bottom. 
   If they've only submitted one iteration, you won't see those icons.
```
